File: app/models/bracket.rb
class Bracket < ActiveRecord::Base
  has_many :teams
  has_many :rounds
  
  def as_json options
    {
      id: self.id,
      teams: self.teams.map{ |t| t.id },
      rounds: self.rounds.map{ |r| r.id },
      title: self.title
    }
  end
end

File: app/assets/javascripts/brakkit/models/bracket.js

//= require ./record

Brakkit.Bracket = Brakkit.Record.extend({
  title : "",
  teams : [],
  rounds : [],
  committed : false,
  urlRoot : '/brackets',
  attributes : function(){
    var self = this;
    return {
      bracket : {
        title : self.get('title')
      }
    };
  }.property('title', 'teams')
});
